qa/rpc-tests/keypool.py

```diff
@@ -1,15 +1,16 @@
 #!/usr/bin/env python3
 # Copyright (c) 2014 The Bitcoin Core developers
+# Copyright (c) 2020 The Zcash developers
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or https://www.opensource.org/licenses/mit-license.php .
 
 # Exercise the wallet keypool, and interaction with wallet encryption/locking
 
-# Add python-bitcoinrpc to module search path:
+# Add python-zcashrpc to module search path:
```

Comment:
I don't understand the comment.   I'm not sure what it refers to.

Here's a chunk of `git annotate` output:

```
6c37f7fd78      (Wladimir J. van der Laan       2014-06-16 14:45:32 +0200       8)# Add python-bitcoinrpc to module search path:
aff0bf7fa1      (Daira Hopwood  2017-06-20 21:09:33 +0100       9)
aff0bf7fa1      (Daira Hopwood  2017-06-20 21:09:33 +0100       10)from test_framework.authproxy import JSONRPCException
aff0bf7fa1      (Daira Hopwood  2017-06-20 21:09:33 +0100       11)from test_framework.util import check_json_precision, initialize_chain,
\
aff0bf7fa1      (Daira Hopwood  2017-06-20 21:09:33 +0100       12)    start_nodes, start_node, stop_nodes, wait_bitcoinds, bitcoind_processes
aff0bf7fa1      (Daira Hopwood  2017-06-20 21:09:33 +0100       13)
6c37f7fd78      (Wladimir J. van der Laan       2014-06-16 14:45:32 +0200       14)import os
6c37f7fd78      (Wladimir J. van der Laan       2014-06-16 14:45:32 +0200       15)import sys
6c37f7fd78      (Wladimir J. van der Laan       2014-06-16 14:45:32 +0200       16)import shutil
6c37f7fd78      (Wladimir J. van der Laan       2014-06-16 14:45:32 +0200       17)import tempfile
6c37f7fd78      (Wladimir J. van der Laan       2014-06-16 14:45:32 +0200       18)import traceback
6c37f7fd78      (Wladimir J. van der Laan       2014-06-16 14:45:32 +0200       19)
```

It seems like @laanwj originally placed the comment before `import os`.   This doesn't make its meaning obvious to me.

Reply:
I suppose the conservative thing to do is leave it untouched.

Reply:
It seems that I have to rewrite history to restore it to its original state.   This is only a problem because reviewers will then need to re-review.

On the bright side I can also rebase atop the most recent master.

Since people have already done review-work I will not reset the comment to the original commit until I get approval from reviewers.

I believe that the options are:

(1)  revert the comment to the original text, but update the committer to me.  
(2)  rewrite the commit history such that the original text is comitted, and attributed to the original author
(3)  don't revert the change

Since I don't understand the comment, I think reverting to the original author is the best solution, but then there's a caveat about review work.

Reply:
Digging a little deeper into the commit history:

```
git annotate master keypool.py | grep python-bitcoinrpc && git annotate master test_framework/test_framework.py | grep python-bitcoinrpc && git annotate master test_framework/util.py | grep python-bitcoinrpc
6c37f7fd78      (Wladimir J. van der Laan       2014-06-16 14:45:32 +0200       8)# Add python-bitcoinrpc to module search path:
e8097f7df1      (Gavin Andresen 2014-07-08 12:07:23 -0400       8)# Add python-bitcoinrpc to module search path:
356cfe8306      (Gavin Andresen 2014-02-26 16:31:18 -0500       10)# Add python-bitcoinrpc to module search path:
```

Reply:
OK...   here's the line the comment almost certainly originally referred to:

https://github.com/bitcoin/bitcoin/blob/356cfe83065aa4f120707f026c10c83f1e62be89/qa/rpc-tests/util.py#L8

Then the comment got copied around, so I vote we remove it entirely (all three instances).

Reply:
OK, this comment has been removed, which resolves this conversation.

Reply:
This is python-bitcoinrpc: https://github.com/jgarzik/python-bitcoinrpc

As far as I know, there is no python-zcashrpc. The dependency is still on python-bitcoinrpc.

Reply:
The URL I posted above shows the code that updates the search path with `python-bitcoinrpc`, but that code is no-longer present, therefore the comment refers to code that's not there.   Therefore I deleted the comment.

Now that I think about it, I'm going to check whether `os` and `sys` are used elsewhere in that module.

Reply:
Ok, they're both used elsewhere.

Reply:
I believe this is resolved (by deleting the comment that references deleted code).